docker/errors.py

```diff
@@ -127,8 +127,14 @@ def __init__(self, container, exit_status, command, image, stderr):
         self.command = command
         self.image = image
         self.stderr = stderr
-        msg = ("Command '{}' in image '{}' returned non-zero exit status {}: "
-               "{}").format(command, image, exit_status, stderr)
+
+        if stderr is None:
+            msg = ("Command '{}' in image '{}' returned non-zero exit "
+                   "status {}").format(command, image, exit_status, stderr)
+        else:
+            msg = ("Command '{}' in image '{}' returned non-zero exit "
+                   "status {}: {}").format(command, image, exit_status, stderr)
```

Comment:
I decided to check how it would be rendered with `None` and found it a bit unfriendly:

    Command 'echo Hello World' in image 'e9aa60c60128' returned non-zero exit status 42: None 

For me it is not clear what does `None` mean in this message.